playFeature/build.gradle

```diff
@@ -42,9 +43,16 @@ dependencies {
   implementation kotlin_core.values()
   implementation junit_test.values()
   implementation compose.values()
+  implementation dagger2.values()
+  kapt dagger2_kapt.values()
+  implementation navigation.values()
 
   implementation project(':domain')
   implementation project(':core')
+  implementation project(':apiDI')
+  implementation project(':dataSourceDI')
```

Comment:
redundant dependencies, moreover you mixed arch layers, brrrrr.... Read few more articles about clean arch or Robert Martin's book

Reply:
dublicate probably because of merge conflict